lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -148,6 +148,9 @@ def __init__(self, namespace: str):
         self.client = Client()
         self.namespace = namespace
 
+    def delete_pod(self, pod: str) -> None:
```

Comment:
I don't think this belongs in the multus library.

Reply:
Yes, it does not belong here. This is draft. I will push another version.

Reply:
Done